mobile-ys/src/app/app.component.ts

```diff
@@ -1,15 +1,20 @@
 import { Component } from '@angular/core'
+import { Store } from '@ngrx/store';
+import { AuthService } from './core/services/auth.service';
+import { Logout } from './core/state/app.action';
 
 @Component({
     selector: 'app-root',
     templateUrl: 'app.component.html',
     styleUrls: ['app.component.scss'],
 })
 export class AppComponent {
-    isLogged: boolean = true
-    constructor() {}
+    isLogged: boolean 
+    constructor(private authService: AuthService, private store: Store<any>) {
+        this.isLogged = this.authService.loggedIn();
+    }
 
     logOut = () => {
```

Comment:
I can't find this button in the interface.

Reply:
mmm sorry, but i don't get it.

Reply:
Mirá te muestro, cuando me logeo, no se ve el botón de Logout.

![imagen](https://user-images.githubusercontent.com/64869535/139599145-a27e9462-b635-49a9-ba36-3ab56d801705.png)